README.md

```diff
@@ -129,11 +129,11 @@ rather a JavaScript object that provides methods to open terminals and run comma
 ### 'atom-xterm' service v2.0.0
 
 The `atom-xterm` service provides the
-[openTerminal()](https://github.com/bus-stop/x-terminal/blob/2a7762b6d29abdc017af17c320b2e548cd14e4a9/src/x-terminal.js#L273) method. The `openTerminal()` method behaves just like Atom's
+[openTerminal()](https://github.com/bus-stop/x-terminal/blob/master/src/x-terminal.js#L262) method. The `openTerminal()` method behaves just like Atom's
```

Comment:
it should be a permalink otherwise if the file changes and the function signature is no longer on line 262 than it will be wrong again

```suggestion
[openTerminal()](https://github.com/bus-stop/x-terminal/blob/42ddb71523a393178cacabca17b38d681b87f292/src/x-terminal.js#L262) method. The `openTerminal()` method behaves just like Atom's
```